### PR TITLE
 feature/1988_space-around-footer

### LINKED ISSUE #1988: Remove space around footer in storybook
Screenshot 2020-10-06 at 08 18 47
Screenshot 2020-10-06 at 08 18 36

Just a small design improvement. In my opinion it would look better without the margin/padding.

### PR DESCRIPTION
**Pull request by raphael-suter**
Fixes #1988 

# Merge Checklist:
- [x] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [x] Tests are sufficient.
- [x] Modifications still work in IE.
- [x] Documentation is up to date.
- [x] Changelog is up to date.
- [x] Typescript typings for properties are up to date.
- [x] Designers approved changes or no approval needed.

### CODE REVIEW

File: src/other/pages/guides/getting-started/story.jsx [hunk 1]
```diff
@@ -157,38 +126,35 @@ export default AXAButtonReact;`}
...
-        {callout(
-          'github-mark.png',
-          'Code of Conduct',
-          'We as the Pattern Library Team commited ourselves to a set of rules, responsibilities and prudent practices.',
-          'https://github.com/axa-ch/patterns-library/blob/develop/CODE_OF_CONDUCT.md',
-          'READ MORE'
-        )}
-
+        <PLCallout
+          className="callout__first-element"
```
Comment (01231) [inline src/other/pages/guides/getting-started/story.jsx]: what about `getting-started__callout-first? So it would be a bit more bem-like.

File: src/components/00-materials/story.scss [hunk 2]
```diff
@@ -1,9 +1,5 @@
-@import '../../other/pages/utils/accessory-story.scss';
 @import '../../other/pages/utils/canvas-content-padding.scss';
 
 @include canvas-padding;
 @include content-padding;
-
-@include accessory-story-content;
```
Comment (01231) [inline src/components/00-materials/story.scss:7]: why was this deleted?

Comment (raphael-suter) [reply src/components/00-materials/story.scss:7]: Icons and colors don't have a footer and therefore don't need a bottom padding.

**Review by 01231 — APPROVED** [review]
Very Nice. Thanks for cleaning up!
